app/src/main/res/values-uk/strings.xml

```diff
@@ -19,59 +19,70 @@
     <string name="select_folder">Обрати директорію</string>
 
     <string name="search_hint">Пошук нотаток</string>
-    <string name="share_string">Поділитися з...</string>
-    <string name="share">Поділитися</string>
+    <string name="share_string">Надіслати до...</string>
+    <string name="share">Надіслати</string>
```

Comment:
New one means 'Send'. As for me it is better to keep previous one.

Reply:
I think translation here is the matter of preference and 'Share' have different translations in different apps (Amaze, NextCloud and others).
I preferred to use 'Send' (to app/recipient) since I consider 'Share' more as 'social network oriented' action.

Reply:
@cl0ne Makes sense.